packages/circuit-ui/components/TopNavigation/TopNavigation.tsx

```diff
@@ -87,7 +100,10 @@ export function TopNavigation({
         isLoading={Boolean(isLoading)}
       >
         {links && <UtilityLinks links={links} />}
-        <ProfileMenu {...profileMenu} user={user} />
+        {user && <ProfileMenu {...profileMenu} user={user} />}
+        {items && items.reverse().map(({ title, ...props }) => (
```

Comment:
**question**: Why is there a need to reverse the items? Can't I expect the caller to pass the items in the order they want them to be shown?

Reply:
The way the component is structured at the moment the items are rendered left to right but are right-aligned. To me it seemed more natural to render them left-to-right (given the right alignment of the items) in which case I would expect the first item in the list to be the right-most, etc. I have no strong opinion on this, so happy to remove the `reverse`.